ddtrace/internal/gitmetadata.py

```diff
@@ -124,3 +124,15 @@ def clean_tags(tags):
     tags.pop(COMMIT_SHA, None)
 
     return tags
+
+
+def update_tags(tags):
```

Comment:
would rename method to indicate better what it does.
for example: add_git_tags(tags).